docs/content/Installation/postgres-operator-installer/_index.md

```diff
@@ -50,4 +50,40 @@ the Crunchy PostgreSQL Operator that you are installing.
 The image pull policy needs to be defined for your job.
 In most cases this should be updated to `IfNotPresent`.
 
+### Running the pgo-client
+#### pgo-client container
+The pgo-client image can be installed along side of the PostgreSQL Operator by
+enabling the `pgo_client_container_install` option in the inventory file.
+This image contains the pgo binary and is setup to have access to the
```

Comment:
Ill probably go in and add back-tics around `pgo`. I've noticed this in a couple places